ExportData.cs

```diff
@@ -20,39 +20,27 @@ namespace Tracker
 {
     public static class ExportData
     {
-        private static string user;
-        private static string computerName;
-        private static string geo_x;
-        private static string geo_y;
-        private static string dynamoVersion;
-        private static string revitVersion;
-        private static string ip;
-        private static string filename;
+        internal static string user;
+        internal static string computerName;
+        internal static string dynamoVersion;
+        internal static string revitVersion;
+        internal static string ip;
+        internal static string latlng;
+        internal static string city;
+        internal static string country;
+        internal static string filename;
         private static string date;
 
-        internal static IList<IList<object>> Export(string filename, string dynamoversion)
+        internal static IList<IList<object>> Export()
         {
             IList<IList<object>> export = new List<IList<object>>();
 
-            user = Environment.MachineName;
-            computerName = Environment.UserName;
+            // Set the date
+            // @todo "hh" is incorrectly returning values in the afternoon. It should be "14" not "2" etc for any hour after midday
             date = DateTime.Now.ToString("yyyy-MM-dd hh:mm:ss");
 
-            WebClient client = new WebClient();
```

Comment:
Web requests can be slow & behave unexpectedly. Also, unlike `user name` and other fields that could change between events, this is not likely to change, so it's a prime candidate to be cached (such as storing it in a `static` prop).

Reply:
Agreed! Hopefully this is resolved with the new static Data class.  Continue discussion in issue https://github.com/teamtreedyn/Binoculars/issues/18 Therefore marking as resolved.